### PR TITLE
Remove dependency on `talon_plugins`

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`talon_plugins` has been removed in the latest beta. As far as I know there is no replacement to check if we are zoomed in or not.

Fixes https://github.com/talonhub/community/issues/1637

### CODE REVIEW

File: plugin/mouse/mouse.py [hunk 1]
```diff
@@ -38,8 +37,7 @@
 class Actions:
     def zoom_close():
         """Closes an in-progress zoom. Talon will move the cursor position but not click."""
-        if eye_zoom_mouse.zoom_mouse.state == eye_zoom_mouse.STATE_OVERLAY:
-            actions.tracking.zoom_cancel()
+        actions.user.deprecate_action("2024-12-26", "user.zoom_close")
```
Comment (auscompgeek) [inline plugin/mouse/mouse.py:40]: Can we point users to call `tracking.zoom_cancel` directly instead?

Comment (AndreasArvidsson) [reply plugin/mouse/mouse.py:40]: That is what I'm doing in the Talon file. `deprecate_action` have no argument for replacement. I was kinda leaning towards maybe adding that.

**Review by phillco — APPROVED** [review]

File: plugin/mouse/mouse.py (hunk 1 above)
Comment (AndreasArvidsson) [reply plugin/mouse/mouse.py]: Optional replacement argument added

File: core/deprecations.py [hunk 2]
```diff
@@ -170,9 +170,11 @@ def deprecate_action(time_deprecated: str, name: str):
         id = f"action.{name}.{time_deprecated}"
 
         deprecate_notify(id, f"The `{name}` action is deprecated. See log for more.")
+        replacement_msg = f' Instead, use: "{replacement}".' if replacement else ""
 
         msg = (
             f"The `{name}` action is deprecated since {time_deprecated}."
+            f"{replacement_msg}"
             f' See {os.path.join(REPO_DIR, "BREAKING_CHANGES.txt")}'
```
Comment (auscompgeek) [inline core/deprecations.py:178]: I guess we want an entry in BREAKING_CHANGES.txt?

**Review by auscompgeek — APPROVED** [review]
Just need docs, otherwise LGTM

File: core/deprecations.py (hunk 2 above)
Comment (AndreasArvidsson) [reply core/deprecations.py:179]: True. Done

**Review by phillco — APPROVED** [review]

**Pull request merged**